src/base/logger.cpp

```diff
@@ -72,7 +72,7 @@ void Logger::freeInstance()
 void Logger::addMessage(const QString &message, const Log::MsgType &type)
 {
     QWriteLocker locker(&m_lock);
-    const Log::Msg msg = {m_msgCounter++, QDateTime::currentMSecsSinceEpoch(), type, message.toHtmlEscaped()};
+    const Log::Msg msg = {m_msgCounter++, QDateTime::currentMSecsSinceEpoch(), type, message};
```

Comment:
why this again?

Reply:
I think this was done to support html formatting else where ( n LogListWidget), this is not needed anymore, if not removed here, had to do unescape in model or delegate, what do we prefer?

Reply:
It was made to avoid some attacks. IIRC calling `toHtmlEscaped()` here would mess up the saved log.txt right? If so, it is not preferable, we should not do that and escape it at model.

Reply:
Sorry, I can't clearly understand what you're talking about...
IMO, it's incorrect to escape here.

Reply:
disregard my comment.

Reply:
Should I remove these changes and unescape at Model or leave the changes?

Reply:
keep the the current changes. no need to make further changes.

Reply:
You should keep your changes here.